mev_inspect/compound_liquidations.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, List
+from web3 import Web3
+from mev_inspect.schemas.classified_traces import ClassifiedTrace
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.schemas.classified_traces import Protocol
+from mev_inspect.historical_price import get_erc20_token_decimals
+from mev_inspect.abi import get_raw_abi
+import os
+
+rpc = os.getenv("RPC_URL")
+
+if rpc is None:
+    raise RuntimeError("Missing environment variable RPC_URL")
+else:
+    w3 = Web3(Web3.HTTPProvider(rpc))
```

Comment:
instead of instantiating `w3`'s globally, there's one created once in `inspect_block` that should be passed down through `get_cether_liquidations`